lib/views/backend/spree/admin/payments/source_forms/_paypal_braintree.html.erb

```diff
@@ -1,10 +1,12 @@
 <% id = payment_method.id %>
+<% radio_buttons_use_name = Gem::Version.new(Spree.solidus_version) < Gem::Version.new('1.3') %>
+<% radio_button_id = radio_buttons_use_name ? 'braintree' : payment_method.id %>
```

Comment:
Since these views are only put into Rails’ loadpath, if `backend` is available, we could make two versions of this file and decide in `engine.rb` which one to pick.